.travis.yml

```diff
@@ -121,7 +121,9 @@ matrix:
 
 before_install: pip3 install --user toml $extra_pip
 
-script: make -j4 && make -j4 -C tests && make check
+script:
+  - make -j4 && make -j4 -C tests && make check
+  - readelf --debug-dump=info src/InfoStream.o | head
```

Comment:
This looks to be left over from some debugging

Reply:
You're absolutely right.  It was left over from trying to find out what version of Dwarf is being used on TravisCI.  Unfortunately, it was the same version that is used on my laptop as well, so I don't yet know why my laptop fails and TravisCI does not.

I'll remove it.  Thank you.